war/src/main/less/modules/section.less

```diff
@@ -1,6 +1,10 @@
+:root {
+  --section-padding: 2rem;
+}
+
 .jenkins-section {
   border-top: 2px solid var(--panel-border-color);
-  padding: calc(2rem) 0;
+  padding: var(--section-padding) 0;
```

Comment:
Not sure if it's this that's causing it but spacing is very off on a new job screen (e.g freestyle)

Some items are touching each other and some are very far apart

before:

![image](https://user-images.githubusercontent.com/21194782/142161626-2ad57fb4-483c-4e3d-bc4f-fbb6f5c2fe93.png)

after:

![image](https://user-images.githubusercontent.com/21194782/142161657-00d27468-dafa-4781-bc01-6e48c46aadaf.png)

Reply:
Checkbox padding (whilst finicky) should be a lot better now - would you mind rechecking this?